build.py

```diff
@@ -38,3 +42,50 @@ def initialize(project):
 
     project.get_property('filter_resources_glob').extend(
         ['**/c_bastion/__init__.py'])
+
+
+def docker_execute(command_list, logger):
+    """ Run and tail a docker command. """
+    import sh
+    try:
+        running_command = sh.docker(command_list, _iter=True)
+        for line in running_command:
+            logger.info(line.strip())
+    except KeyboardInterrupt:
+        logger.info("Requested to terminate running docker command.")
+        running_command.process.terminate()
+
+
+@task
+def docker_run(project, logger):
+    logger.info("Running the docker image.")
+    AUTH_URL = os.environ['AUTH_URL']
+    docker_execute(['run',
+                    '-p', '8080:8080',
+                    '-p', '2222:2222',
+                    '-e', 'AUTH_URL={0}'.format(AUTH_URL),
+                    '--name', docker_name,
+                    'c-bastion:latest',
+                    ], logger)
+
+
+@task
+@depends("package")
+@depends("clean")
+def docker_build(project, logger):
+    logger.info("Building the docker image.")
+    docker_execute(['build', '-t', 'c-bastion', '.'], logger)
```

Comment:
And then do a `docker tag ... to give it the `latest` tag after the system test is successful or even only when you push it to dockerhub